dist/selleckt.js

```diff
@@ -613,21 +617,33 @@ _.extend(SellecktPopup.prototype, {
 
             e.stopPropagation();
 
-            if($target.is(itemClass) || $target.is(searchInputClass)){
+            if ($target.is(itemClass) || $target.is(searchInputClass)) {
                 return;
             }
             self.close();
         });
     },
 
+    _getPopupMaxHeight: function() {
+        var $itemsList;
+
+        if (!this.maxHeightPopupPositioning) {
+            return this.$popup.outerHeight();
+        }
+
+        $itemsList = this.$popup.find('.' + this.itemslistClass);
+
+        return this.$popup.outerHeight() + parseInt($itemsList.css('max-height')) - $itemsList.height();
```

Comment:
Please always pass the radix parameter to  `parseInt`, e.g. `parseInt(something, 10)` - I'm not even sure if this is relevant here, but anytime good practice to do so. 

[this](https://davidwalsh.name/parseint-radix) shows the weird cases.